docs/UserGuide.adoc

```diff
@@ -248,5 +291,7 @@ e.g. `find James Jake`
 * *Select* : `select INDEX` +
 e.g.`select 2`
 * *History* : `history`
+* *Remark*: `remark INDEX [r/REMARK]`
 * *Undo* : `undo`
 * *Redo* : `redo`
+* *Favorite*: `favorite INDEX`
```

Comment:
Remember to include features you've planned for v2.0 in the user guide as well. You can label them as "coming in v2.0" or something similar.